multinet.rst

```diff
@@ -1,12 +1,122 @@
+as a social network connecting the various Revolutionary War figures to the
+social clubs they belonged to.
+
+To create a workspace to hold the materials for this dataset, use the ``NEW
+WORKSPACE`` button appearing near the top of the left sidebar. A dialog box will
+appear asking for the name of the new workspace. Fill this in with ``boston``
+and click on the ``CREATE WORKSPACE`` button. This will create the workspace and
+activate it.
+
+In the main panel, you should see two empty columns, headed by ``Networks`` and
+``Tables`` respectively. These columns will track all of the data that comes to
+live in this workspace.
+
+**2. Upload Node Tables**
+
+Multinet largely thinks of attribute data in tabular terms. That is, Multinet
+can ingest data formatted as (among other thigns) CSV files, storing it as a
+*table* of *rows*, each row containing the same types of attributes in the same
+order. Every node table must have a special attribute called ``_key``, which
+contains a value unique to that row within the table it is part of.
+
+We will begin by uploading separate tables to account for the *people* and
+*clubs* represented in this dataset. First, download :download:`members.csv
+</files/members.csv>`; this file contains the names of several historical
+figures important during the American Revolution, along with some attributes.
+Next, click on the plus icon to the right of the ``Tables`` column to upload a
+file and create a table. Specify the ``members.csv`` file, note that the system
+autofills in the name of the table to create, and then click the ``CREATE``
+button.
+
+This network contains two *node types*---one to encode the people, and another
+for the clubs they belong to. Multinet recognizes nodes encoded in different
+tables as being of different node types, so download :download:`clubs.csv
+</files/clubs.csv>` and upload it to a ``clubs`` table, using the same procedure
+as above.
+
+You should now see both a ``members`` and a ``clubs`` table listed. You can
+click on each of these to see the rows of each table. Use the browser's back
+button to return to the workspace view.
+
+**3. Upload an Edge Table**
+
+Next we want to specify the topology of this network---that is, the relationship
+between clubs and members as specified by membership. We need a special table
+that encodes these memberships, using special attributes ``_from`` and ``_to``.
+These attributes will always refer to rows from other tables in the workspace,
+using a *node ID*, which is formed by joining the name of the table and the key
+of the row with a slash. For instance, the John Adams row has a key value of
+``7``; its node ID is therefore ``members/7``.
+
+You can think of the rows of such a table as a directed links connecting two of
+the nodes from somewhere in the workspace. To see this in action, download
+:download:`/files/membership.csv`, and upload it as a table. Because this table
```

Comment:
```suggestion
:download:`membership.csv</files/membership.csv>`, and upload it as a table. Because this table
```

Reply:
Excellent catch. I did it a bit differently in 3faceb1e183e17b1d9251fbba6587fc30b137660.